tests/test_errors.py

```diff
@@ -13,7 +13,8 @@ def test_no_errors():
 
 
 def test_HTTPError():
-    resp = mock.Mock(status_code=404)
+    resp = mock.Mock(status_code=404,
+                     content="File not found")
```

Comment:
We should test that this body is not correctly set in the exception. Also this project is using single quotes for strings.

I will take care of both small changes and merge this. Thanks for the bug fix!